src/components/ContentSidebar/ActivitySidebar.js

```diff
@@ -438,29 +459,59 @@ class ActivitySidebar extends React.PureComponent<Props, State> {
             activityFeedError,
             currentUserError,
         } = this.state;
+
+        // Since this is the first render, we know that ActivityFeed will finally be lazy loaded in!
```

Comment:
This is where things get a little messy. `React.lazy()` ONLY begins loading when the first `render()` occurs. We need to start a timer when this happens. After it DOES happen, then we need to start listening for the AF to be in the loading state AND/OR the full renderable state. 

Note that this logic dictates that if a full render occurs before the load state is rendered, the time it took to get to a load state render will not be recorded. This can occur when `currentUser` and `feedItems` API requests come back before the ActivityFeed component JS is loaded

Reply:
Great point